### PR TITLE
Moo: moo moo moo

### LINKED ISSUE #57: Test issue - ignore
Moo :cow:

### PR DESCRIPTION
**Pull request by jodh-intel**
Moo moo moo moo, moo; moo? MOO!

Fixes #57.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Let's not include this one in the weekly summary eh? :)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-[![Build Status](https://travis-ci.org/kata-containers/proxy.svg?branch=master)](https://travis-ci.org/kata-containers/proxy)
-[![codecov](https://codecov.io/gh/kata-containers/proxy/branch/master/graph/badge.svg)](https://codecov.io/gh/kata-containers/proxy)
-
-# Kata Containers Proxy
+```
```
Comment (egernst) [inline README.md]: I think you could use larger moo here. :interrobang:

Comment (jodh-intel) [reply README.md]: What like?

```
MMMMMMMM               MMMMMMMM                                  
M:::::::M             M:::::::M                                  
M::::::::M           M::::::::M                                  
M:::::::::M         M:::::::::M                                  
M::::::::::M       M::::::::::M   ooooooooooo      ooooooooooo   
M:::::::::::M     M:::::::::::M oo:::::::::::oo  oo:::::::::::oo 
M:::::::M::::M   M::::M:::::::Mo:::::::::::::::oo:::::::::::::::o
M::::::M M::::M M::::M M::::::Mo:::::ooooo:::::oo:::::ooooo:::::o
M::::::M  M::::M::::M  M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M   M:::::::M   M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M    M:::::M    M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M     MMMMM     M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M               M::::::Mo:::::ooooo:::::oo:::::ooooo:::::o
M::::::M               M::::::Mo:::::::::::::::oo:::::::::::::::o
M::::::M               M::::::M oo:::::::::::oo  oo:::::::::::oo 
MMMMMMMM               MMMMMMMM   ooooooooooo      ooooooooooo   
```

**Comment by jodh-intel** [comment]
MOOOoooo..... . :boom:

**Pull request closed without merging**